kdenlive/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.1"
+
+services:
+  server:
+    image: linuxserver/kdenlive:421.12.3
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - SUBFOLDER=/
+    volumes:
+      - ${APP_DATA_DIR}/data/config:/config
+    ports:
```

Comment:
Once the app proxy is added, we can remove this block

Reply:
Are you talking about the `ports` section?

Reply:
Yes; the `ports` section for the `server` service can be dropped once we use the app proxy.

Reply:
pushed changes for this